### PR TITLE
OCPBUGS-11821: check ocp version and export CNF_TEST_IMAGE variable with appropriate…

### PR DESCRIPTION
**Pull request by mrniranjan**
… cluster version

### CODE REVIEW

**Comment by ffromani** [comment]
/approve

ack for the general direction

File: hack/run-functests.sh [hunk 1]
```diff
@@ -21,6 +22,10 @@ if [ -n "${IMAGE_FORMAT}" ]; then
   LATENCY_TEST_RUN="true"
 fi
 
+if [[ -z "${CNF_TEST_IMAGE}" ]]; then
+    ocp_version=${OC_TOOL} get clusterversion -o jsonpath='{.items[].status.desired.version}{"\n"}' | awk -F "." '{print $1"."$2}'
```
Comment (ffromani) [inline hack/run-functests.sh:26]: please add
```
echo "autodetected cluster version: ${ocp_version}"
```

**Comment by yanirq** [comment]
@mrniranjan   I think we need to add this to https://github.com/openshift/cluster-node-tuning-operator/blob/master/hack/run-update-testing.sh as well

File: hack/run-functests.sh [hunk 2]
```diff
@@ -21,6 +22,12 @@ if [ -n "${IMAGE_FORMAT}" ]; then
   LATENCY_TEST_RUN="true"
 fi
 
+if [[ -z "${CNF_TEST_IMAGE}" ]]; then
```
Comment (yanirq) [inline hack/run-functests.sh:25]: I see 2 occurrences of the env var:
- CNF_TEST_IMAGE
- CNF_TEST**S**_IMAGE
Which one is the right one ?

File: hack/run-latency-testing.sh [hunk 3]
```diff
@@ -15,6 +16,12 @@ if ! which tput &> /dev/null 2>&1 || [[ $(tput -T$TERM colors) -lt 8 ]]; then
   NO_COLOR="-noColor"
 fi
 
+if [[ -z "${CNF_TEST_IMAGE}" ]]; then
```
Comment (yanirq) [inline hack/run-latency-testing.sh:19]: Same comment as above:



I see 2 occurrences of the env var:

    CNF_TEST_IMAGE
    CNF_TESTS_IMAGE

    Which one is the right one ?

File: hack/run-functests.sh (hunk 2 above)
Comment (ffromani) [reply hack/run-functests.sh:25]: the code consumes `CNF_TESTS_IMAGE` (with the S)

**Comment by yanirq** [comment]
@mrniranjan please squash all the commits into one

File: hack/run-latency-testing.sh (hunk 3 above)
Comment (mrniranjan) [reply hack/run-latency-testing.sh]: CNF_TESTS_IMAGE is the right one

**Comment by ffromani** [comment]
the latest run were done against HEAD _before_ the PR which should fix them. Best would be rebase again.

**Comment by ffromani** [comment]
/retest-required

infra issue

**Pull request merged**